src/wasm/kotlin/java/arcs/RuntimeClient.kt

```diff
@@ -11,23 +11,23 @@
 
 package arcs
 
-expect fun utf8ToStringImpl(bytes: ByteArray): String
-expect fun stringToUtf8Impl(str: String): ByteArray
-
-fun ByteArray.utf8ToString(): String = utf8ToStringImpl(this)
-fun String.stringToUtf8(): ByteArray = stringToUtf8Impl(this)
-
+//expect fun utf8ToStringImpl(bytes: ByteArray): String
+//expect fun stringToUtf8Impl(str: String): ByteArray
+//
+fun ByteArray.utf8ToString(): String =  throw NotImplementedError()
```

Comment:
These can be implemented by calling out to the browser TextEncoder/Decoder APIs, and with conversion functions from TypedArray (Uint8Array) to ByteArray.